docs/index.md

```diff
@@ -16,9 +16,57 @@ To see RedisGraph in action, visit [Demos](https://github.com/RedisLabsModules/r
 
 ## Quickstart
 
-1. [Build](#building)
-2. [Start](#loading-redisgraph-into-redis)
-3. [Use from any client](#using-redisgraph)
+1. [Docker](#docker)
+2. [Build](#building)
+3. [Start](#loading-redisgraph-into-redis)
+4. [Use from any client](#using-redisgraph)
+
+## Docker
+
+To quickly tryout RedisGraph, lunch an instance using docker:
+
+```sh
+docker run -p 6379:6379 -it --rm redislabs/redisgraph
+```
+
+Once loaded you can interact with RedisGraph using redis-cli.
+
+Here we'll quickly create a small graph representing a subset of motorcycle riders and teams taking part in the MotoGP league,
```

Comment:
I think you should add a title or something else here to distance it from the "Docker" paragraph